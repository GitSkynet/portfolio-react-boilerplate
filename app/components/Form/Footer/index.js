import styled from 'styled-components';

const Footer = styled.div`
  padding: 16px 0 8px;
  text-align: center;
`;

export default Footer;
