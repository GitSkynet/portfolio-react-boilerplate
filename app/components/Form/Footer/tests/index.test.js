import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import Footer from '../index';

describe('<Footer />', () => {
  it('should render and match the snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
