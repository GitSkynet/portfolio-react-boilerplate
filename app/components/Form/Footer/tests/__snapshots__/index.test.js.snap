// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render and match the snapshot 1`] = `
.c0 {
  padding: 16px 0 8px;
  text-align: center;
}

<div
  className="c0"
/>
`;
